src/pke/unittest/UnitTestCoexistingCryptocontexts.cpp

```diff
@@ -58,7 +58,7 @@ class UTGENERAL_CRYPTOCONTEXTS : public ::testing::Test {
 TEST_F(UTGENERAL_CRYPTOCONTEXTS, coexisting_ckks_cryptocontexts) {
     // Setup crypto context 1
     CCParams<CryptoContextCKKSRNS> parameters1;
-    parameters1.SetMultiplicativeDepth(4);
+    parameters1.SetMultiplicativeDepth(2);
```

Comment:
why 4 to 2?

Reply:
the test fails with multDepth=4 if NATIVE_SIZE=128. as i remember correctly, we would need to tune up more parameters if we simply did not set the depth to 2. now it works for any integer size